prime-router/src/main/kotlin/azure/SettingsFacade.kt

```diff
@@ -408,4 +469,119 @@ class ReceiverAPI
     description,
     transport
 ),
-    SettingAPI
\ No newline at end of file
+    SettingAPI
+
+class ReceiverSerializer constructor(t: Class<ReceiverAPI>) :
```

Comment:
Can we add some kdoc for these classes please?

Reply:
> Can we add some kdoc for these classes please?

cust serializers removed - no longer needed